### ISSUE TITLE
Anything LLM

### ISSUE THREAD

**Issue by computersrmyfriends** [FlowiseAI/Flowise#2415]
Is it possible to integrate with Anything LLM?

**Comment by kirkjaa** [comment]
Yes this would be a definite plus for programmable agents indeed.

**Comment by zfusx** [comment]
we wrote a serve to bride anythingllm and flowise ai ,
"A middleware server that creates an OpenAI-compatible API layer for Flowise, allowing you to use Flowise chatflows with any application or library designed to work with OpenAI's chat completions API."
anyone would struggle with link anythingllm to flowise ai ,can have a try:
https://github.com/zfusx/toble-bridge
